packages/Liquid-Core.package/LiquidPoll.class/instance/addAnswer..st

```diff
@@ -1,3 +1,3 @@
 as yet unclassified
 addAnswer: aPollAnswer
-	answerSets add: aPollAnswer.
\ No newline at end of file
+	(open)ifTrue: [answerSets add: aPollAnswer].
```

Comment:
[Sand] Are the braces required here? I remember getting scolded  for stuff like this in SWA